app/templates/account/csv_settings.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'layouts/base.html' %}
+{% import 'macros/form_macros.html' as f %}
+
+{% block content %}
+    <div class="ui grid container">
+        <div class="eight wide computer sixteen wide mobile centered column">
+            <h2 class="ui dividing header">Set CSV Import Settings</h2>
+            {{ f.render_form(form) }}
+
+        </div>
+    </div>
+{% endblock %}
+
```

Comment:
Daaaaamn alex, back at it again with those newlines (but seriously cut down on the newlines)